### PR TITLE
Added models and started routes.

### PR DESCRIPTION
**Pull request by SusannaRiccardi**
The models are complete and the routes have been started.

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: recipes/config.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  // MongoDB
+  // Url of the Mongodb server
+  mongoUrl: "mongodb://localhost/",
+  // Database name
+  mongoDbName: "atelierbeats-dev",
```
Comment (triglian) [inline recipes/config.js]: change the name to `forkit-dev`

File: recipes/npm-debug.log [hunk 2]
```diff
@@ -0,0 +1,45 @@
+0 info it worked if it ends with ok
```
Comment (triglian) [inline recipes/npm-debug.log]: remove this file with a new commit. After the commit, update your `.gitignore` to ignore `npm-debug.log`

File: recipes/package.json [hunk 3]
```diff
@@ -1,17 +1,30 @@
...
-}
\ No newline at end of file
+  "name": "recipes",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "start": "node ./bin/www"
+  },
+  "dependencies": {
+    "adaro": "^1.0.4",
```
Comment (triglian) [inline recipes/package.json]: Since you're using `jade` remove `adaro` which is for dust templates

File: recipes/routes/index.js [hunk 4]
```diff
@@ -1,9 +1,25 @@
...
+
 var express = require('express');
 var router = express.Router();
 
 /* GET home page. */
 router.get('/', function(req, res, next) {
-    res.render('layout', {});
+  // TODO: Deal with index route.
+  res.render('layout', {});
+  // res.render('index', { title: 'Express' });
```
Comment (triglian) [inline recipes/routes/index.js]: remove commented code before merging

File: recipes/routes/recipes.js [hunk 5]
```diff
@@ -1,9 +1,77 @@
...
+  newRecipe.save(function(err, saved) {
+    if (err) {
+      throw err
+    }
+    res.json(saved);
+  })
+})
+
+// Function to decide how to render the view.
+function htmlOrJson(req, res, data, viewName) {
```
Comment (triglian) [inline recipes/routes/recipes.js]: remove if you don't use it. You can always go back to the commit that removed it and restore the code

**Review by triglian — CHANGES_REQUESTED** [review]

File: recipes/app.js [hunk 6]
```diff
@@ -1,6 +1,7 @@
 var express = require('express');
```
Comment (triglian) [inline recipes/app.js:1]: Whichever variable doesn't change reference should be `const`. If it does, then use `let`. There is seldom an usecase for `var`

File: recipes/routes/index.js [hunk 7]
```diff
@@ -1,9 +1,25 @@
+// This is just something to start. As soon as we have a more complete website, the routes will be adapted.
+
 var express = require('express');
```
Comment (triglian) [inline recipes/routes/index.js]: See what I wrote about `var` above

File: recipes/routes/recipes.js [hunk 8]
```diff
@@ -1,9 +1,105 @@
...
+    }
+    // Add href.
+    for (let obj of found) {
+      addLinks(obj);
+    }
+
+    let recipes = {'recipes' : found}
+    console.log(recipes.recipes);
+    res.render('recipes', {title : "Recipes", message : recipes});
+    // htmlOrJson(req, res, recipes, 'recipes');
```
Comment (triglian) [inline recipes/routes/recipes.js]: remove commented code.

**Review by triglian — APPROVED** [review]

**Pull request merged**